pkg/flare/archive.go

```diff
@@ -1001,7 +1007,13 @@ func getArchivePath() string {
 	dir := os.TempDir()
 	t := time.Now().UTC()
 	timeString := strings.ReplaceAll(t.Format(time.RFC3339), ":", "-")
-	fileName := strings.Join([]string{"datadog", "agent", timeString}, "-")
+	logLevel := config.Datadog.GetString("log_level")
+	fileName := ""
+	if len(logLevel) != 0 {
+		fileName = strings.Join([]string{strings.ToUpper(logLevel), "datadog", "agent", timeString}, "-")
+	} else {
+		fileName = strings.Join([]string{strings.ToUpper(logLevel), "datadog", "agent", timeString}, "-")
+	}
```

Comment:
3 small issues here:
- Both condition blocks execute the same code
- We should lowercase the log level string to match the other hardcoded strings
- I thing for UX (like bash auto-completion), we would want to retain the same prefix and put the logging level after that (`datadog-agent-<level>-<timestamp>` or `datadog-agent-<timestamp>-<level>`)

Reply:
- The conditions have been updated to what I had intended.
- `datadog-agent-<timestamp>-<level>` is the format which is now in place.
- the log level string added is lowercase